app/models/agents/webhook_agent.rb

```diff
@@ -69,5 +71,9 @@ def validate_options
     def payload_for(params)
       Utils.value_at(params, interpolated['payload_path']) || {}
     end
+
+    def response_message
+      options['response'] || 'Event Created'
```

Comment:
I'd normally say that this should be `options['response'].presence || 'Event Created'`, so that `""` defaults to "Event Created".  However, I think you explicitly want this to be an empty string, right?

We should at least do `interpolated['response'] || 'Event Created'` so that people can use Liquid templating if they want to.